backend/package.json

```
@@ -1,45 +1,25 @@
 {
   "name": "@wm-tools/backend",
   "version": "1.0.0",
-  "type": "module",
-  "description": "Small utility (REST API) that reads and writes configs for the WM Tools to AWS S3",
   "scripts": {
-    "start": "build & node dist/server.js",
-    "build": "pnpm tsc --build tsconfig.build.json && tsc-alias -p tsconfig.build.json",
-    "typecheck": "tsc --noEmit"
+    "dev": "wrangler dev",
+    "build": "tsc --project tsconfig.json",
+    "deploy:cloudflare": "wrangler deploy --compatibility-date 2024-09-23 --compatibility-flag nodejs_compat"
```

Move these to `wrangler.toml`

```suggestion
    "deploy:cloudflare": "wrangler deploy"
```